plugin/openingh.lua

```diff
@@ -5,9 +5,15 @@ vim.g.openingh = true
 
 local openingh = require("openingh")
 
-vim.api.nvim_create_user_command("OpenInGHFile", function()
-  openingh:openFile()
-end, {})
+vim.api.nvim_create_user_command("OpenInGHFile", function(opts)
+  if opts.range == 0 or opts.range == 1 then
+    openingh.openFile()
```

Comment:
I'm not sure why this was previously using the colon notation but since I needed to pass parameters, I'm now using the dot notation.

Reply:
Yes, we don't need to pass the object as the first argument to that method, please change the other call to a dot notation in line 19